tests/test_databases.py

```diff
@@ -800,7 +800,7 @@ async def test_queries_with_expose_backend_connection(database_url):
     """
     async with Database(database_url) as database:
         async with database.connection() as connection:
-            async with database.transaction(force_rollback=True):
+            async with connection.transaction(force_rollback=True):
```

Comment:
The docs for transactions need to be updated too.

Reply:
Sure. Thank you for the fix! (I'm glad it got merged finally).